sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_multimodal/_content_safety_multimodal.py

```diff
@@ -0,0 +1,244 @@
+                            },
+                            {
+                                "type": "image_url",
+                                "image_url": {
+                                    "url": "<image url or base64 encoded image>"
+                                }
+                            }
+                        ]
+                    },
+                    {
+                        "role": "assistant",
+                        "content": "This picture shows an astronaut standing in desert."
+                    }
+                ]
+            }
+        )
+        
+    **Output format**
+    .. code-block:: python
+        {
+            "violence": "Medium",
+            "violence_score": 5.0,
+            "violence_reason": "Some reason",
+            "sexual": "Medium",
+            "sexual_score": 5.0,
+            "sexual_reason": "Some reason",
+            "self_harm": "Medium",
+            "self_harm_score": 5.0,
+            "self_harm_reason": "Some reason",
+            "hate_unfairness": "Medium",
+            "hate_unfairness_score": 5.0,
+            "hate_unfairness_reason": "Some reason"
+        }
+    """
+    
+    def __init__(
+        self, 
+        credential, 
+        azure_ai_project, 
+        parallel: bool = False
+    ):
+        self._parallel = parallel
+        self._evaluators: List[Callable[..., Dict[str, Union[str, float]]]] = [
+            ViolenceMultimodalEvaluator(credential, azure_ai_project),
+            SexualMultimodalEvaluator(credential, azure_ai_project),
+            SelfHarmMultimodalEvaluator(credential, azure_ai_project),
+            HateUnfairnessMultimodalEvaluator(credential, azure_ai_project),
+        ]
+
+    def __call__(
+            self, 
+            *, 
+            conversation, 
+            **kwargs):
+        """
+        Evaluates content-safety metrics for list of messages.
+        :keyword conversation: The conversation contains list of messages to be evaluated. Each message should have "role" and "content" keys.
+        :paramtype conversation: ~azure.ai.evaluation.Conversation
+        :return: The scores for messages.
+        :rtype: Dict
+        """
+        self._validate_conversation(conversation)
+        
+        results: Dict[str, Union[str, float]] = {}
+        if self._parallel:
+            with ThreadPoolExecutor() as executor:
+                futures = {
+                    executor.submit(evaluator, conversation=conversation, **kwargs): evaluator
+                    for evaluator in self._evaluators
+                }
+
+                for future in as_completed(futures):
+                    results.update(future.result())
+        else:
+            for evaluator in self._evaluators:
+                result = evaluator(conversation=conversation, **kwargs)
+                results.update(result)
+
+        return results
+    
+    def _validate_conversation(self, conversation):
```

Comment:
this seems like a good utility function to have available to multiple evaluators. Can you move this to a util module?